bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,54 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
-#----alsa install
-sudo apt-get install -y alsa-base alsa-utils
+#----install official requirements
+sudo apt-get install -y alsa-base alsa-utils libasound2-dev
+
+#----install missing pigpio in Raspbian Lite (the command npm install pigpio will be exec by package.json)
+sudo apt-get install pigpio
+
+#----install git and download tjbot
+if [ ! -d "${TJBOT_FOLDER}/recipes/conversation" ]; then
+    sudo apt-get install -y git
+
+    rm -Rf /home/pi/git/tjbot
+    git clone https://github.com/ibmtjbot/tjbot.git /home/pi/git/tjbot
+
+    TJBOT_FOLDER='/home/pi/git/tjbot'
+fi
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+echo "path: $PWD"
+
+sudo npm install --unsafe-perm -g node-gyp
```

Comment:
The target is moving :). Up till now I thought we were discussing what happens in `/home/pi/tjbot/recipes/conversation`, but now we're talking about `/home/pi/tjbot/bootstrap/tests`.

Are you 100% sure you were logged on as user pi and that the file attached above is the output of `npm install` and NOT the output of `sudo npm install`? Because it looks like the output of `sudo npm install` which is a completely different thing.

What I see when `npm install` is called in `/home/pi/tjbot/bootstrap/tests` can be seen at https://gist.github.com/fivdi/f7a45e595d3039cb0c5d631425bb080d

At the end if the gist `sudo node test.servo.js` is successfully called.